ui/components/src/model/units/bytes.test.ts

```diff
@@ -28,7 +28,7 @@ const BYTES_TESTS: UnitTestCase[] = [
   {
     value: -1234,
     unit: { kind: 'Bytes', abbreviate: false, decimal_places: 4 },
-    expected: '-1,234 bytes',
+    expected: '-1,234.0000 bytes',
```

Comment:
All these changes show "decimal_places" becoming a setting that means "use exactly X decimal places".